packages/scoutgame-ui/src/components/projects/ProjectsPageContainer.tsx

```diff
@@ -0,0 +1,26 @@
+import { getSession } from '@packages/nextjs/session/getSession';
+import { safeAwaitSSRData } from '@packages/nextjs/utils/async';
+import { getUserScoutProjects } from '@packages/scoutgame/projects/getUserScoutProjects';
+
+import { ProjectsPage } from './ProjectsPage';
+
+export default async function ProjectsPageContainer() {
+  const session = await getSession();
+  const scoutId = session.scoutId;
+
+  if (!scoutId) {
+    return null;
+  }
+
+  const allPromises = [getUserScoutProjects({ userId: scoutId })] as const;
+
+  const [error, data] = await safeAwaitSSRData(Promise.all(allPromises));
+
+  if (error) {
+    return null;
```

Comment:
this seems like the wrong way to handle an error... what does the user see?

Reply:
We are logging an error inside `safeAwaitSSRData`. There are other places where we are using this pattern so I just followed it but I agree this will show nothing to the user if there is an error. We should clarify that in some sort of alert